frontend/lib/evictionfree/declaration-builder/covid-impact.tsx

```diff
@@ -1,81 +1,91 @@
 import React from "react";
 import { CheckboxFormField } from "../../forms/form-fields";
 import { SessionUpdatingFormSubmitter } from "../../forms/session-updating-form-submitter";
+import { li18n } from "../../i18n-lingui";
 import { MiddleProgressStep } from "../../progress/progress-step-route";
 import { EvictionFreeCovidImpactMutation } from "../../queries/EvictionFreeCovidImpactMutation";
 import { Accordion } from "../../ui/accordion";
 import { ProgressButtons } from "../../ui/buttons";
 import Page from "../../ui/page";
 
 const FinancialHardshipAccordion: React.FC<{}> = () => (
-  <Accordion question="What does “financial hardship” mean?">
+  <Accordion question={li18n._(t`What does “financial hardship” mean?`)}>
     <p>
-      This means you are unable to pay your rent or other financial obligations
-      under the lease in full or obtain alternative suitable permanent housing
-      because of one or more of the following:
+      <Trans id="evictionfree.financialHardshipExplainer1">
+        This means you are unable to pay your rent or other financial
+        obligations under the lease in full or obtain alternative suitable
+        permanent housing because of one or more of the following:
+      </Trans>
     </p>
     <ul className="jf-space-below-2rem">
-      <li>
-        Significant loss of household income during the COVID-19 pandemic.
-      </li>
-      <li>
-        Increase in necessary out-of-pocket expenses related to performing
-        essential work or related to health impacts during the COVID-19
-        pandemic.
-      </li>
-      <li>
-        Childcare responsibilities or responsibilities to care for an elderly,
-        disabled, or sick family member during the COVID-19 pandemic have
-        negatively affected your ability or the ability of someone in your
-        household to obtain meaningful employment or earn income or increased
-        your necessary out-of-pocket expenses.
-      </li>
-      <li>
-        Moving expenses and difficulty you have securing alternative housing
-        make it a hardship for you to relocate to another residence during the
-        COVID-19 pandemic.
-      </li>
-      <li>
-        Other circumstances related to the COVID-19 pandemic have negatively
-        affected your ability to obtain meaningful employment or earn income or
-        have significantly reduced your household income or significantly
-        increased your expenses.
-      </li>
-      <li>
-        To the extent that you have lost household income or had increased
-        expenses, any public assistance, including unemployment insurance,
-        pandemic unemployment assistance, disability insurance, or paid family
-        leave, that you have received since the start of the COVID-19 pandemic
-        does not fully make up for your loss of household income or increased
-        expenses.
-      </li>
+      <Trans id="evictionfree.financialHardshipExplainer2">
```

Comment:
Oy, just FYI one unfortunate thing about this explainer is that it is _not_  technically a copy-paste from the actual hardship declaration form: the form is in the _first person_, so I had to modify it to be in the _second person_.  I hope this isn't too much work for Johanna. 😞 

That said, I wonder if one quicker way to translate it might be to take the exact same text from the Spanish version of the hardship declaration and convert it to be in the second-person, rather than translating this section "from scratch".

Reply:
oooh great thanks for flagging. I'll keep that in mind when I hand things off